patterns/cta-events-list.php

```diff
@@ -61,25 +61,25 @@
 			<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--accent-6);border-top-width:1px;padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)"><!-- wp:group {"layout":{"type":"constrained"}} -->
 				<div class="wp-block-group">
 					<!-- wp:heading {"level":3} -->
-					<h3 class="wp-block-heading">“Stories, historias, iсторії, iστορίες”</h3>
+					<h3 class="wp-block-heading"><?php echo esc_html_x( '“Stories, historias, iсторії, iστορίες”', 'Sample heading in four languages', 'twentytwentyfive' ); ?></h3>
```

Comment:
> “Stories, historias, iсторії, iστορίες”

This text exists in several patterns, some with context and some without. It might be better to standardize it to either `esc_html_x` or `esc_html_e`.